flink-runtime/src/test/java/org/apache/flink/runtime/shuffle/NettyShuffleUtilsTest.java

```diff
@@ -99,6 +100,8 @@ public void testComputeRequiredNetworkBuffers() throws Exception {
                         maxRequiredBuffersPerGate,
                         sortShuffleMinParallelism,
                         numSortShuffleMinBuffers,
+                        false,
+                        tieredStoreExclusiveBuffers,
```

Comment:
The last two commits need to be tested, especially the `GateBuffersSpecTest`.

Reply:
Fixed.